Makefile

```
@@ -3,6 +3,10 @@ GOCMD = go
 GOTEST = $(GOCMD) test 
 WASIRUN_WRAPPER := $(CURDIR)/scripts/wasirun-wrapper
 
+# Coverage
+COVERAGE_REPORT := coverage.out
```

Let's make sure that file is created in an ignored dir:

```suggestion
COVERAGE_REPORT := build/coverage.out
```

---

Done.